.github/workflows/update-types.yaml

```diff
@@ -0,0 +1,55 @@
+on: # event list
+  push: # on every push to master (basically when PR's are merged into master) / or the target branch
+    branches:
+      - master
+
+env: # environment variables (available in any part of the action)
+  NODE_VERSION: 16
+
+jobs: # list of things to do
+  linting:
```

Comment:
should this be `update-types`?

Reply:
Yes, should be `update-types`.

Re
> Is there a way to prevent the merge if there are type diffs? Or maybe make the build fail?
> It would be nice to not have a separate PR/commit bc of continuous integration. But if it's not possible, this totally works.

Yes, we can run it as a check when creating a new PR. 

My reasoning for doing it like this is that we can make it automatically merge or push directly to master, and we don't have to bother with manually updating the types.

This auto PR/commit won't change the output files, so, CI/CD shouldn't be triggered maybe?

Anyway, this is only how I viewed it, but I realize it might be wrong. I'm up to changing it! just let's clarify in slack.